flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CorrelateITCase.scala

```diff
@@ -389,6 +389,25 @@ class CorrelateITCase extends StreamingTestBase {
     assertThat(sink.getAppendResults.sorted).isEqualTo(expected.sorted)
   }
 
+  @Test
+  def testL1eftJoinPartialProjectWithEmptyOutput(): Unit = {
```

Comment:
```suggestion
  def testLeftJoinPartialProjectWithEmptyOutput(): Unit = {
```

Reply:
btw why is it called EmptyOutput?

Reply:
shouldn't the name of the method contain `CrossJoin`?

Reply:
yep, makes sense rename the test method with more suitable name